clients/gtest/gbmv_batched_gtest.cpp

```diff
@@ -72,11 +64,8 @@ const vector<char> transA_range = {
     // 'C',
 };
 
-// number of gemms in batched gemm
-const vector<int> batch_count_range = {
```

Comment:
You could remove size 1 and change 5 to batch 2 if you really just want to ensure the looping over batches.  This is what I set for some bad_arg batch sizes in rocblas, just ensure a hard coded 1 doesn't get used somewhere.